pos-print/build.gradle

```diff
@@ -72,7 +71,7 @@ dependencies {
     exclude module: 'guice'
     exclude module: 'guice-servlet'
     exclude module: 'guice-multibindings'
-    exclude module: 'jackson-databind'
+//    exclude module: 'jackson-databind'
```

Comment:
The commented line need to be removed if jackson is no longer going to be ignored.